DeDRM_plugin/ignobleepub.py

```diff
@@ -338,79 +338,79 @@ def cli_main():
 
 def gui_main():
     try:
-        import Tkinter
-        import Tkconstants
-        import tkFileDialog
-        import tkMessageBox
+        import six.moves.tkinter
+        import six.moves.tkinter_constants
+        import six.moves.tkinter_filedialog
+        import six.moves.tkinter_messagebox
```

Comment:
from six.moves import tkinter, tkinter_constants, tkinter_filedialog, tkinter_messagebox

Reply:
I realized only now that `ignobleepub.py` is a modified copy of `ineptepub.py`. `ineptepub.py` has already been converted to Python 3 and uses the `import six.moves.tkinter` syntax as I do (since this is what `python-modernize` produces). Changing this only in `ignobleepub.py` would make them drift apart even further.